coderd/inboxnotifications.go

```diff
@@ -60,6 +66,7 @@ func convertInboxNotificationResponse(ctx context.Context, logger slog.Logger, n
 // @Param targets query string false "Comma-separated list of target IDs to filter notifications"
 // @Param templates query string false "Comma-separated list of template IDs to filter notifications"
 // @Param read_status query string false "Filter notifications by read status. Possible values: read, unread, all"
+// @Param format query string false "Define the output format for notifications title and body. Possible values: plaintext, markdown"
```

Comment:
```suggestion
// @Param format query string false "Define the output format for notifications title and body" enums(plaintext,markdown)
```
Not sure if you can use enum with string, but this seems more like what we'd want?

Reply:
Seems to work well, thanks was not aware of this one 🚀